docs/subgroups.md

```diff
@@ -25,4 +25,7 @@ The language-specific subgroups are
   * Saúl Cabrera (Shopify)
 * Go - organized by
   * Achille Roussel (Stealth Rocket)
-  * Jiaxiao (Joe) Zhou (Microsoft)
+  * Jiaxiao (Joe) Zhou (Microsoft) 
+* C#/.net - organized by
+  * Timmy Silesmo - (nor²)
+  * Scott Wayne
```

Comment:
```suggestion
  * Scott Wayne
  * Danilo (Dan) Chiarlone (Microsoft)
```

Reply:
Hi Dan, the organizing members list is just the people responsible for scheduling, running, recording the notes for the subgroup, and reporting back to the main SIG.

You don't need to be on of the organizing members to participate or to indicate support for the subgroup being made.

If you'd like specifically to be an organizing member instead of one of the proposed ones, can you give your reasoning here?

Reply:
Sounds good - I just figured I'd add my name to the list because C# wit-bindgen support is something I'm working on together w/ a few other folks at MSFT, so feel free to ignore my suggested comment.

Reply:
I guess the way the subgroups doc is written, there's nothing against having three organizing members. There's just only so much work to do, so it doesn't necessarily make sense to have more from a pragmatic perspective. That said, it may still make sense to add you in your capacity as a representative of the Microsoft work since the OMs are informally co-chairs. If @silesmo and @yowl are interested, that could be worth doing.

Reply:
I'm easy either way.  Obviously don't want to duplicate effort if Microsoft are already working on wit-bindgen?  Sounds like we have two repos doing the same thing.  Although I am focusing on NativeAOT and perhaps Microsoft are concentrating on Mono?

Reply:
Language-specific subgroups are a great place to coordinate and avoid duplicate work in a given ecosystem. The JavaScript/TypeScript subgroup has been having some great discussions about how to unify tooling. Regardless of who end up being the organizing members, I hope all three of you are involved in the subgroup and help unify the C#/.net tooling work.

The question for this PR is whether to add Dan Chiarlone as an organizing member, and after thinking about it more I think that's a good idea. So, if @silesmo is fine with it, I'd suggest that we accept Dan's change here to add himself as an OM.

Reply:
I'm happy to add you as well @danbugs.